app/controllers/application_controller.rb

```diff
@@ -354,6 +354,7 @@ def autologin
     make_logged_in_user_available_to_everyone
     track_last_time_user_made_a_request
     block_suspended_users
+    clear_admin_mode_for_nonadmins
```

Comment:
I don't see a problem with this, but I don't understand why it's needed or why it's part of this PR.